acct_mgt_test.py

```diff
@@ -643,3 +716,52 @@ def test_project_user_role(acct_mgt_url, auth_opts):
                 is True,
                 "user " + "test0" + str(user_number) + "unable to be deleted",
             )
+
+
+def test_quota(acct_mgt_url, auth_opts):
+    """This tests quota support on the microserver"""
+    # 1) Create a project
+    project_name = "test-003"
+    if not oc_resource_exist("project", "Project", project_name):
```

Comment:
Even visual studio code provides this when you hover over a function which does as much as keyword arguments would do:

![Screen Shot 2021-12-15 at 5 21 20 PM](https://user-images.githubusercontent.com/23174942/146274092-bd3d78af-cef6-4495-8c8d-74b5df14c5b6.png)

I am not certain if adding keyword arguments would be worth it.  I am fairly consistent with naming, the oc_ prefix for any command that is run using oc, the "_exists" postfix for any any command that tries to see if it exists and if so returns a True.  In this case the command would be 'oc get project [project_name]' and would compare what is returned to see if "kind"= "Project"

Reply:
I meant to add keywords to the caller, the function definition is fine.

Reply:
I understood what you meant, I hovered over the function call and up pops the function definition.

I don't think adding keywords to the function parameters will improve readability.  You will still have to look at the function definition to see how it is defined as many function keywords would be ambiguously defined.  I would want to have the same ordering of parameters as that reads better - and this something that happens naturally by design with positional arguments.  Any optional parameters are always last with positional arguments, not always true with keyword arguements.

Just consider oc_resource_exist:

looking at where it called from:

      oc_resource_exist("project", "Project", project_name)

Just from the naming and where it is in the code, would you agree that it looks like it is returning if the project exists?

Based on my familiarity with oc, project is used in the oc command line, "Project" is compared with json that is returned and project_name could either be the name in the json or the project name.    

if it had keywords it would read like:

    oc_resource_exists(resource="project", kind="Project", name=project_name)

Here, if you are not familiar with what oc passes back, you probably would not know what resource, or kind are.  name is a little easier to guess at - but you would still need to look at the function definition in order to know what it is doing.

I don't see how much further you are ahead with keyworded arguments in this case.  Perhaps if more of the arguments were optional, but that would probably add complexity to the function itself.  I don't see the added complexity for keyboarding the arguments to be worth the added complexity.

Reply:
okay, I see your point now. thanks.